fixtures/american-dj/vbar-pak.json

```diff
@@ -0,0 +1,413 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "VBAR PAK",
+  "categories": ["Color Changer", "Pixel Bar"],
+  "meta": {
+    "authors": ["Ken Harris"],
+    "createDate": "2022-03-23",
+    "lastModifyDate": "2022-03-23"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/8659/vbar_pak.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/vbar-pak"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=bah_MlTImIg"
+    ]
+  },
+  "physical": {
+    "dimensions": [560, 63, 90],
+    "weight": 1.7,
+    "power": 21,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "5× 4W RGBA LED"
+    },
+    "lens": {
+      "degreesMinMax": [40, 40]
+    }
+  },
+  "availableChannels": {
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [16, 31],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [32, 47],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [48, 63],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [64, 79],
+          "type": "ColorPreset",
+          "comment": "Amber"
+        },
+        {
+          "dmxRange": [80, 95],
+          "type": "ColorPreset",
+          "comment": "Red & Green",
+          "colors": ["#ffff00"]
```

Comment:
Is this one mixed color for the whole fixture, or different colors for different pixels? Then, it'd be `"colors": ["#ff0000", "#00ff00"]`. (same for the following capabilities)

Reply:
I assume it's one color for the whole fixture, with all red and all green LEDs at full.

This fixture doesn't seem to provide pixel-level addressing, and this is the simplest mode.  But I haven't actually tested this.